### PR TITLE
Readme updates!

### PR DESCRIPTION
**Pull request by Andy-Python-Programmer**
Readme updates!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
 <p align="center">Minimal Birb programming language!</p>
 
 # Contribution
 
 - Join the discord - https://discord.gg/9dq6YB2
-- Clone the repository
-- Navigate to the new directory and install birb by running:
+- Make a fork of this repo.
+- Clone the fork.
+- Navigate to the cloned directory and build birb by running:
```
Comment (Calamity210) [inline README.md:15]: Can we revert the changes to this line

File: README.md [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
 ```shell
 $ pub get
 $ dart2native lib/birb.dart
 ```
-
 - When ready, open and pull request and request a review from me - Calamity210.
 
+More info about contributing can be found here: https://birbolang.web.app/docs/contributing
```
Comment (Calamity210) [inline README.md:22]: The websites link is directly below, id prefer to remove this

**Review by Calamity210 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Andy-Python-Programmer) [reply README.md:15]: Sure

**Review by Calamity210 — APPROVED** [review]
